paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/CompactAction.java

```diff
@@ -74,6 +77,12 @@ public CompactAction withPartitions(List<Map<String, String>> partitions) {
         return this;
     }
 
+    static Options getOptions(MultipleParameterTool params) {
+        Options options = new Options();
+        params.toMap().forEach((k, v) -> options.set(k, v));
+        return options;
+    }
+
```

Comment:
How about using `catalog-conf k=v`?
Like:
https://paimon.apache.org/docs/master/how-to/cdc-ingestion/#synchronizing-tables

Reply:
That's a good idea, I'm going to modify it in this format